_src/css/_header.scss

```diff
@@ -7,17 +7,17 @@
   background-image: url('../assets/taina.png');
   background-repeat: no-repeat;
   background-position: bottom center;
```

Comment:
@felipprodrigues you just forgot to add these repeated background rules in a separate media query.